bylaws/009-funding.md

```diff
@@ -0,0 +1,57 @@
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes requires a bylaw change vote. 
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][].
+
+## Fiscal hosting
+
+For handling money and fiscal matters, the PHP-FIG MUST use [Open Collective](https://opencollective.com/) as a platform and [Open Source Collective](https://opencollective.com/opensource) as a fiscal host.
+
+All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's OpenCollective account. They SHALL NOT use those permissions in a discretionary manner, except as stated explicitly in this bylaw.
+
+All funds MUST be collected into the Open Collective PHP-FIG account, so that any tax considerations will be handled automatically by the fiscal host, and every financial decision MUST be published and publicly available at the bottom of this document in the appropriate table and through Open Collective's budgetary tools.
+
+## Raising money
+
+PHP-FIG SHALL NOT solicit donations from any party, especially from private individuals; all exceptions to this rule MUST be listed below.
+
+### Through OpenCollective
+
+Direct contributions MUST NOT be accepted through Open Collective, to avoid private/personal donations; the collective home page MUST be set up in a way that:
+ * all contribution sections and features are hidden and/or disabled, if possible;
+ * explicit wording will be visible reminding visitors of the PHP-FIG stance in the matter (see "Scope and objectives").
+
+### Through TideLift
+
+Contributions SHALL be accepted through [TideLift](https://tidelift.com/). All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's TideLift account, which MUST be configured to pay out to the Open Collective account. 
+
+## Spending money
+
+Funds donated to the PHP-FIG MUST be used solely for non-personnel, operating expenses.  PHP-FIG SHALL NOT pay individual contributors to PHP-FIG standards or other personnel, like Core Committee members, Secretaries, Project Representatives or working groups.
+All expenses MUST be approved by an Approval Vote of the Core Committee.  The expense MUST include whether it is one time or recurring and at which frequency.  All expenses MUST be justified by their contribution to the mission of PHP-FIG.
```

Comment:
Proposed addition to accommodate for price changes:
```
Secretaries MAY request Implicit Approval from the Core Committee for changing the approved amount for previously accepted recurring expenses.
```

Reply:
Do we want to put a cap on this rule? Something like a percentage, 10% maybe?

I would rephrase that as:

> Secretaries MAY request an Implicit Approval to change the approved sum for a recurring expense, if it's a decrease or it does not exceed a 10% increase from the previously approved amount; higher increases or any other type of change MUST go through a regular Approval Vote.

Do you like this form? I also added an "any other type of change" so that i.e. a provider change has to be explicitly approved too.

Reply:
I have no strong opinion on this. My intended wording would become:
> All expenses MUST be approved by an Approval Vote of the Core Committee.  The expense MUST include whether it is one time or recurring and at which frequency.  All expenses MUST be justified by their contribution to the mission of PHP-FIG.  Secretaries MAY request Implicit Approval from the Core Committee for changing the approved amount for previously accepted recurring expenses.

I think that is already clear enough that Implicit Approval is only applicable to price changes. If you'd want to limit the amount, you'd also have to specify a period. Alternatively, wording like this could be used:

> All expenses MUST be approved by an Approval Vote of the Core Committee.  The expense MUST include whether it is one time or recurring and at which frequency.  All expenses MUST be justified by their contribution to the mission of PHP-FIG.  In response to supplier price changes for previously accepted recurring expenses, Secretaries MAY request Implicit Approval from the Core Committee to raise the approved amount.

Reply:
IMO the Implicit Approval raising should then be limited to recurring expenses.  For one-offs, if the CC approved $50, and the Secretaries then decide to spend $100, what was the point of the original CC vote?

Reply:
@Crell A good argument, but Implicit Approval requires notification and a 7 day period for objection.  An example (though I'm not sure the FIG would ever hit it as It probably wouldn't be aligned with mission, but) would be some sort of customized merch order.  These are often priced per piece and the final order can be over/under those pieces by a certain percent for various reasons I've never taken the time to understand.  In that case, implicit approval would be reasonable.

Reply:
10% sounds reasonable to me, especially with US inflation hovering around 6-8%

Reply:
Point of order: US inflation is now around 3%.  It has come back down rather dramatically in the past year. 😄

Reply:
10% cap added! I did it only for recurring expenses, since I expect that to be solved by Secretaries' due diligence (and a bit of overhead) for one-shot expenses.